src/models/ObjectLevelCan.ts

```diff
@@ -5,13 +5,20 @@ type ObjectLevelCan<TPermissions extends string> = {
   [key in TPermissions]: boolean
 }
 
-export type BasicPermissionsObjectLevelCan = ObjectLevelCan<BasicPermissions>
-export type BasicRunnablePermissionsObjectLevelCan = ObjectLevelCan<BasicPermissions | 'run'>
+export const BasicPermissions = ['create', 'read', 'update', 'delete'] as const
+export const RunnablePermissions = [...BasicPermissions, 'run'] as const
 
-export function createObjectLevelCan<T extends Record<string, boolean>>(): T {
+export type BasicPermissionsObjectLevelCan = ObjectLevelCan<typeof BasicPermissions[number]>
+export type BasicRunnablePermissionsObjectLevelCan = ObjectLevelCan<typeof RunnablePermissions[number]>
+
+export function createObjectLevelCan<T extends Record<string, boolean>>(knownProperties: typeof BasicPermissions | typeof RunnablePermissions): T {
+  const set = new Set(knownProperties)
   return new Proxy({} as T, {
-    get() {
-      return true
+    get(___, property) {
```

Comment:
super nit: just because we haven't used the underscore skip pattern anywhere else

```suggestion
    get(target, property) {
```

Reply:
I feel like... I do very much prefer not seeing this at a glance 
<img width="522" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/eb218722-067f-493b-85dc-d984789fd3db">


vs 

<img width="238" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/a7bc4594-0c8f-45fe-a98a-fe56e742bebe">

Reply:
like vs code seems to know what i'm telling it so it feels like a common enough convention to me

Reply:
weird, never noticed vscode did magic with underscores for skips. The greyed out argument doesn't bother me. In fact I kinda prefer that because I can see its not used. And eslint is smart enough to not flag that its not used because it was needed to access the second property

Reply:
idk, we could start using underscores but we should updated our linting rules to allow a single underscore. I've just never liked this pattern because it ends up with thinks like
```typescript
fn(_, __, ___, thingIWant)
```

Reply:
oh the grey'ed out to me makes me immediately want to go in and delete it. especially with the popover which isn't literally a warning but I associate it w/ linting warns

Reply:
Well its a super nit so not going to block this PR lol. If you wanna start introducing that pattern lets update eslint to allow single and double underscores 👍

Reply:
<img width="344" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/5c4bf866-f657-4a4f-adc6-6110c2339b0a">

vs code says the "quick fix" is to _prefix it with an underscore_

Reply:
<img src="https://media.giphy.com/media/3o85xIO33l7RlmLR4I/giphy.gif" />

Reply:
Lol 🤷